hey-discount/src/main/java/Ad.java

```diff
@@ -0,0 +1,177 @@
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+class Review {
```

Comment:
Please always avoid declaring two different classes in one file unless one of the classes is a nested (internal) one. This confuses a lot